JS/carlot-events.js

```diff
@@ -0,0 +1,60 @@
+var CarLot = (function(newCarLot) {
+    var addListener = [];
+    newCarLot.activateEvents = function() {
+        addListener = document.getElementsByClassName("click-event");
+        var search = document.getElementById('search-input');
+        for (var i = addListener.length - 1; i >= 0; i--) {
+            
+            //8) When you click on one of the car elements, change the width of the border to a higher value,
+            // and change the background color to any other color of your choosing.
+            addListener[i].addEventListener("click",function(e){
+
+                if (e.target.classList.contains('col-md-4')){
+                    document.getElementById(e.target.id).classList.toggle("triggered");
```

Comment:
If I double click on an element the typing feature doesn't work because the class is gone.  You'll need to figure out a way to remove the class when a different div is clicked, not every other time this div is clicked.